runtime/moonbeam/src/lib.rs

```diff
@@ -99,6 +99,13 @@ use sp_runtime::{
 	Perquintill, SaturatedConversion,
 };
 use sp_std::{convert::TryFrom, prelude::*};
+use xcm::{
+	v3::{AssetId as XcmAssetId, Location},
+	IntoVersion, VersionedAssetId, VersionedAssets, VersionedLocation, VersionedXcm,
+};
+use xcm_config::AssetType;
+use xcm_fee_payment_runtime_api::Error as XcmPaymentApiError;
+use xcm_primitives::UnitsToWeightRatio;
```

Comment:
This should probably be moved to `runtime/common/src/apis.rs`.

Is there a reason for using `xcm::v3` instead of `xcm::v4`?

Reply:
I think these imports cannot be moved to a common place inside `runtime/common/src/apis.rs`. The reason is because we need to import them in the place where the code generated by the `impl_runtime_apis_plus_common` macro will be inserted, and this place ends up being each runtime file.

Other solution would be to import these files inside each function of the `XcmPaymentApi` that makes use of them, but I don't like this too much. What do you think?

> Is there a reason for using xcm::v3 instead of xcm::v4?

Yes. This is because our `asset-manager` pallet is still configured in `v3`, so to avoid any kind of compatibility issues between versions, is better to use `v3` until we migrate the pallet to `v4`.

Reply:
Understood. About the imports, there should be a better way and have them in a single place (maybe at the top of the macro). But I am also fine with this

Reply:
Sometime it's better to duplicate the code rather than making it more complex, we already duplicate some type definitions and imports for each runtime, it help to keep the codebase readable and easy to understand